dlp/snippets/deid_test.py

```diff
@@ -305,3 +305,24 @@ def test_deidentify_with_exception_list(capsys):
 
     assert "gary@example.org" not in out
     assert "jack@example.org accessed record of user: [EMAIL_ADDRESS]" in out
+
+
+def test_deidentify_table_replace_with_info_types(capsys):
+    table_data = {"header": ["age", "patient", "happiness_score", "factoid"],
+                  "rows": [
+                      ["101", "Charles Dickens", "95", "Charles Dickens name was a curse invented by Shakespeare."],
+                      ["22", "Jane Austen", "21", "There are 14 kisses in Jane Austen's novels."],
+                      ["90", "Mark Twain", "75", "Mark Twain loved cats."]]}
+
+    deid.deidentify_table_replace_with_info_types(
+        GCLOUD_PROJECT,
```

Comment:
```suggestion
        GOOGLE_CLOUD_PROJECT,
```

Reply:
I suppose there is nothing to change here

Reply:
@leahecole Line 26 has
`GCLOUD_PROJECT = os.getenv("GOOGLE_CLOUD_PROJECT")`

Are you suggesting that we should rename the variable to GOOGLE_CLOUD_PROJECT?
Please clarify!

Reply:
I was suggesting a variable rename, butit's fine - it seems to match other snippets